app/src/main/java/com/rob729/quiethours/Activity/IntroScreen.kt

```diff
@@ -0,0 +1,52 @@
+package com.rob729.quiethours.Activity
+
+import android.content.Intent
+import android.content.SharedPreferences
+import android.os.Bundle
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import com.github.appintro.AppIntro
+import com.github.appintro.AppIntroCustomLayoutFragment
+import com.github.appintro.AppIntroPageTransformerType
+import com.rob729.quiethours.R
+
+class IntroScreen : AppIntro() {
+
+    lateinit var sharedPreferences: SharedPreferences
```

Comment:
StoreSession is being used for storing values in shared preferences.
Used it for the same.